apps/web/src/state/settings/internal.ts

```diff
@@ -0,0 +1,9 @@
+import { proxy } from 'valtio';
+
+export interface SettingState {
+	_id?: string;
+	_rev?: string;
+	isPrivate?: boolean;
+}
+
+export const settingState = proxy<SettingState>({});
```

Comment:
It may be great to have a default setting, we don't need to handle undefined problems.